docs/InstallationConfiguration.md

```diff
@@ -77,47 +51,10 @@ class Config
 
 ## LEAF_Request_Portal
 
-Copy `globals.php` from the configuration files provided they will look like:
+Create `LEAF_Request_Portal/sources/Config.php` and add the following code into the file:
 
 ```php
 <?php
-if (!defined('PRODUCT_NAME')) define('PRODUCT_NAME', getenv('PRODUCT_NAME'));
-if (!defined('VERSION_NUMBER')) define('VERSION_NUMBER', getenv('PORTAL_VERSION_NUMBER'));
-
-if (!defined('DATABASE_DB_ADMIN')) define('DATABASE_DB_ADMIN', getenv('DATABASE_DB_ADMIN'));
-
-if (!defined('DIRECTORY_HOST')) define('DIRECTORY_HOST', getenv('DATABASE_HOST'));
-if (!defined('DIRECTORY_DB')) define('DIRECTORY_DB', getenv('DATABASE_DB_DIRECTORY'));
-if (!defined('DIRECTORY_USER')) define('DIRECTORY_USER', getenv('DATABASE_USERNAME'));
-if (!defined('DIRECTORY_PASS')) define('DIRECTORY_PASS', getenv('DATABASE_PASSWORD'));
-
-if (!defined('LEAF_NEXUS_URL')) define('LEAF_NEXUS_URL', getenv('APP_URL_NEXUS'));
-if (!defined('HTTP_HOST')) define('HTTP_HOST', getenv('APP_HTTP_HOST'));
-if (!defined('AUTH_URL')) define('AUTH_URL', getenv('APP_URL_AUTH'));
-if (!defined('AUTH_TYPE')) define('AUTH_TYPE', getenv('APP_AUTH_TYPE'));
-if (!defined('CIPHER_KEY')) define('CIPHER_KEY', getenv('APP_CIPHER_KEY'));
-if (!defined('LIB_PATH')) define('LIB_PATH', getenv('APP_LIB_PATH'));
-
-```
-
-Create the following files `sources/DbConfig.php` and `sources/Config.php` and place the following code into them
```

Comment:
I would keep this block for now until we move over to using just the db for configs.

Reply:
I'm in agreement with Shane. Changes to the Config files in both the LEAF_Nexus and LEAF_Request_Portal are coming in the next sprint I believe. When those changes come all the data will be pulled from the db and these classes will be updated like all other classes and won't need to be protected by gitignore. The DB_Config is going to go away completely and will no longer be used.

Reply:
This change will get superseded by this change https://github.com/department-of-veterans-affairs/LEAF/pull/1939